x-pack/plugins/observability_solution/dataset_quality/public/hooks/use_degraded_docs_chart.tsx

```diff
@@ -96,12 +108,15 @@ export const useDegradedDocsChart = ({ dataStream }: DegradedDocsChartDeps) => {
       breakdownFieldName: breakdownDataViewField?.name,
     });
     setAttributes(lensAttributes);
+    if (isBreakdownFieldEcsAsserted) trackDatasetDetailsBreakdownFieldChanged();
   }, [
     breakdownDataViewField?.name,
     euiTheme.colors.danger,
     setAttributes,
     dataStream,
     dataStreamStat?.title,
+    trackDatasetDetailsBreakdownFieldChanged,
```

Comment:
`trackDatasetDetailsBreakdownFieldChanged` makes the flyout chart produce runtime errors. When I refresh  the flyout, I see:
![image](https://github.com/user-attachments/assets/a05dbdc1-ab8a-42e6-befb-98ad0d57eb21)

If I comment this dependency, it doesn't produce errors.